nginx.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: docs
+title: Nginx Configuration
+headline: How to Configure Nginx for Pico
+description: Nginx can be very powerful, but it takes a little extra configuration with Pico - Let's break it down.
+toc:
+  getting-started: Getting Started
+  general-server-configuration: General Server Configuration
+  denying-access-to-picos-internal-files: Denying Access to Pico's Internal Files
+  php-configuration: PHP Configuration
+  url-rewriting:
+    _title: URL Rewriting
+    pico-in-document-root: Pico in Document Root
+    pico-in-a-subfolder:
+      _title: Pico in a Subfolder
+      regex-and-nginxs-processing-order: Regex and Nginx's Processing Order
+  advanced-configuration-tips:
+    _title: Advanced Configuration Tips
+    modular-pico-config: Modular Pico Config
+nav-url: /docs/
+gh_release: v1.0.2
+redirect_from: /nginx.html
```

Comment:
The `redirect_from` plugin is used for BC; this is a new page, thus not necessary.